docs/build/linux-ubuntu.md

```diff
@@ -63,17 +63,33 @@ See [node.md](node.md) for more information.
 
 #### Emscripten
 
-You'll also have to install `emscripten` (version 3.1.44 is known to work):
+You'll also have to install `emscripten`:
 
 ```shell
 git clone https://github.com/emscripten-core/emsdk.git
 cd emsdk
-./emsdk install 3.1.44
-./emsdk activate 3.1.44
-export EMSCRIPTEN_BASE=$(pwd)/upstream/emscripten
+./emsdk install 3.1.58
+./emsdk activate 3.1.58
```

Comment:
What about using the env variable from `minimum-versions.inc.sh`, then we don't have to update the doc when we update the emscripten version.

```suggestion
. /path/to/keyman/resources/build/minimum-versions.inc.sh
./emsdk install ${KEYMAN_MIN_VERSION_EMSCRIPTEN}
./emsdk activate ${KEYMAN_MIN_VERSION_EMSCRIPTEN}
```

Reply:
Only trouble is then the copy-paste approach fails with the /path/to/... bit. So we win some, lose some.